e107_admin/lancheck.php

```diff
@@ -1847,12 +1847,12 @@ function edit_lanfiles($dir1,$dir2,$f1,$f2,$lan)
 			$rowamount = round(strlen($trans['orig'][$sk])/34)+1;
 			$hglt1=""; $hglt2="";
 			if ($trans['tran'][$sk] == "" && $trans['orig'][$sk]!="") {
-				$hglt1="<span class='label label-danger label-important e-tip' title='Missing' >";
+				$hglt1="<span class='label label-danger label-important e-tip' title=".LAN_MISSING.">";//Missing
```

Comment:
It appears title is missing the single quotes around LAN_MISSING. ie. `title='".LAN_MISSING.'" `